### PR TITLE
WIP - trying to make API calls in React

### PR DESCRIPTION
**Pull request by chompasina**
- Adds Axios to make HTTP requests
- Adds Basic styling
- Trying to make API calls, but running into errors so it's commented out for now

Feedback requested on Axios implementation, specifically: 
- How to render multiple components (do they all need to be connected to app.js?)
- How to store API response in local storage
- How to secure API key in React (I'm pretty sure there's not really a way)

### CODE REVIEW

**Comment by chompasina** [comment]
This is what is looks like now: 
screen shot 2016-10-27 at 2 20 26 pm

File: src/App.js [hunk 1]
```diff
@@ -2,19 +2,28 @@ import React, { Component } from 'react';
...
     this.state = { zips: [] };
+    // this.state = { zips: [], results: [] };
   
 }
 componentDidMount() {
   let zipsFromStorage = localStorage.getItem('zips');
   zipsFromStorage = JSON.parse(zipsFromStorage);
 
-  this.setState({zips: zipsFromStorage ? zipsFromStorage : []});
+  this.setState({zips: zipsFromStorage ? zipsFromStorage : [], results: []});
```
Comment (matthewrpacker) [inline src/App.js]: @chompasina I really like the use of the conditional(ternary) operator in this case.  It is easy to understand and makes for a good one line conditional.

File: src/InputSubmission.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ class InputSubmission extends Component {
             onChange={ ( event ) => this.setState({ zip: event.target.value }) }
              /><br/>
 
-          <button onClick={ this.handleClick.bind(this) } >Submit</button>
+          <button id="button" onClick={ this.handleClick.bind(this) } >Submit</button>
```
Comment (matthewrpacker) [inline src/InputSubmission.js]: @chompasina Great job incorporating `.bind`.  I saw that you utilized it elsewhere, too.  Nice work!

File: src/App.js (hunk 1 above)
Comment (chompasina) [reply src/App.js]: Thanks Matt! It's gotten more complicated now, but that was helpful feedback. I'll refactor so it can look like this again. :)

**Pull request merged**